build.gradle.kts

```diff
@@ -51,6 +52,7 @@ dependencies {
 	implementation("org.openapitools:jackson-databind-nullable:$openApiToolsVersion")
 	implementation ("org.springframework.cloud:spring-cloud-starter-openfeign")
 	implementation("org.hibernate.validator:hibernate-validator:$hibernateValidatorVersion")
+	implementation("org.springframework.boot:spring-boot-starter-data-mongodb:$mongoDbVersion")
```

Comment:
you are bringing forward the spring boot version only for the `starter-data-mongodb`. I mean the mongoDbVersion variable doesn't really refer to any particular version (as the name may intend) but to a particular version of spring boot already defined at row 3.

Reply:
My mistake, already solved!